src/components/HomePage/HomePage.js

```diff
@@ -1,32 +1,67 @@
 import React, { Component } from "react";
+import { countryCodes } from "../../Data/CountryCodes";
 import "./HomePage.css";
+import { fetchSearch } from "../../api-calls";
+import { artistData } from "../../Data/mockArtistDataset";
 
 export class HomePage extends Component {
   constructor() {
     super();
     this.state = {
-      bandArtist: "",
-      album: "",
-      track: "",
-      label: "",
+      artist: "",
+      country: "",
+      searchResult: [],
     };
   }
 
   handleChange = (event) => {
     this.setState({ [event.target.name]: event.target.value });
-  }
+  };
 
   submitSearch = (event) => {
     event.preventDefault();
-    console.log(this.state)
-    this.clearInputs()
-  }
+    const convertToQuery = this.state.artist.split(" ").join("+");
+
+    fetchSearch(this.state.artist, this.state.country).then((data) => {
+      if (data.artists.length > 1) {
+        this.setState({ searchResult: data.artists });
+      } else {
+        this.props.handleArtistID(data.artists[0].id);
+      }
+      this.clearInputs();
+    });
+  };
 
   clearInputs = () => {
-    this.setState({bandArtist: "", album: "", track: "", label: ""})
-  }
+    this.setState({ artist: "", country: "" });
+  };
+
+  createArtistsList = () => {
+    const listedArtists = this.state.searchResult.map((artist, index) => {
+      if (artist.score >= 70 && artist["life-span"].begin) {
+        return (
+          <li key={index} onClick={() => this.props.handleArtistID(artist.id)}>
+            {artist.name}: from {artist["life-span"].begin} -{" "}
+            {artist["life-span"].end ? artist["life-span"].end : "present"}
```

Comment:
Are you trying to interpolate or conditionally render info on lines 44-45? If interpolating, can I suggest wrapping the lines in {} and then interpolate your logic within?